app/poros/user.rb

```diff
@@ -0,0 +1,9 @@
+class User
+  attr_reader :id,
+    :email
+
+  def initialize(data)
+    @id = data[:id]
+    @email = data[:attributes][:email]
```

Comment:
User will also have first and last name! Maybe ask Johnny about what Oauth will spit out